app/controllers/pages_controller.rb

```diff
@@ -3,7 +3,7 @@ def home
   end
 
   def about
-    @members = Member.all
+    @members = Member.all.sort_by{ |member| member.member_role}
```

Comment:
We don't enforce an order on member roles, so this `sort_by` call also depends on the order in which we insert roles into the database (which depends on the order in which we insert members into the database).

This doesn't seem mission-critical, so feel free to address it later. But I suggest leaving a
```
TODO(shimmy): Enforce member role order
```
here or in the `MemberRole` model for the future.